modules/search/instant-search/components/search-results.jsx

```diff
@@ -93,15 +86,18 @@ class SearchResults extends Component {
 						) ) }
 					</ol>
 				) }
-				<div className="jetpack-instant-search__search-pagination">
-					{ this.props.hasNextPage && (
-						<ScrollButton
-							enableLoadOnScroll={ this.props.enableLoadOnScroll }
-							isLoading={ this.props.isLoading }
-							onLoadNextPage={ this.props.onLoadNextPage }
-						/>
-					) }
-				</div>
+				{ ! hasResults && <SearchResultsEmpty query={ this.props.query } /> }
+				{ hasResults && (
```

Comment:
```suggestion
				{ hasResults && this.props.hasNextPage && (
```

Perhaps we can combine the conditionals into one clause? I don't see why we'd want to render an empty `.jetpack-instant-search__search-pagination`.